apps/desktop-wallet/src/App.tsx

```diff
@@ -54,16 +57,21 @@ const App = memo(() => {
   return (
     <ThemeProvider theme={theme === 'light' ? lightTheme : darkTheme}>
       <GlobalStyle />
-
       <SplashScreen />
-
       <WalletConnectContextProvider>
         <AppContainer>
           <CenteredSection>
+            {!isWalletUnlocked && (
+              <AnimatedBackground
+                anchorPosition="bottom"
+                opacity={theme === 'dark' ? 0.6 : 0.8}
+                verticalOffset={-100}
+                hiddenOverflow
+              />
+            )}
```

Comment:
There is an issue here. I found a solution. PR coming.


https://github.com/user-attachments/assets/40288d9e-5bb6-4e3d-a498-0630a7cee61f